### PR TITLE
Fix autolink on startup

### PR DESCRIPTION
**Pull request by ericpromislow**
* if there's no prefs file, assume it's a first-run
* found yet another bug related to some binaries being in a bin/
  subdirectory (while minikube isn't)
* fix some of the logic/log messages to give more straightforward output
  when the results are expected

### CODE REVIEW

File: src/config/settings.js [hunk 1]
```diff
@@ -91,6 +91,19 @@ function init() {
   return settings;
 }
 
+async function isFirstRun() {
+  let settingsPath = join(paths.config(), 'settings.json');
```
Comment (mook-as) [inline src/config/settings.js:95]:
```suggestion
  const settingsPath = join(paths.config(), 'settings.json');
```

File: src/config/settings.js [hunk 2]
```diff
@@ -91,6 +91,19 @@ function init() {
...
+  try {
+    await util.promisify(fs.access, fs.constants.F_OK)(settingsPath, fs.constants.F_OK);
```
Comment (mook-as) [inline src/config/settings.js:97]: Nothing in the docs seems to indicate that `util.promisify` can take more than one argument?

(We could also use the [pre-promisified version](https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode) if you want.)

File: src/config/settings.js [hunk 3]
```diff
@@ -91,6 +91,19 @@ function init() {
...
+    return false;
+  } catch(err) {
+    if (err.code !== 'ENOENT') {
+      console.log(`Checking for existence of ${settingsPath}, got error ${err}`);
```
Comment (mook-as) [inline src/config/settings.js:101]: Should we throw here?

**Review by mook-as — APPROVED** [review]

File: src/config/settings.js (hunk 3 above)
Comment (ericpromislow) [reply src/config/settings.js:101]: I don't think it's consequential enough.

**Review by mattfarina — APPROVED** [review]

**Comment by ericpromislow** [comment]
merged in 6355714

**Review by mook-as — APPROVED** [review]

**Pull request merged**